InteractiveSideMenu.xcodeproj/project.pbxproj

```diff
@@ -266,13 +266,13 @@
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				INFOPLIST_FILE = "$(SRCROOT)/Sources/Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
```

Comment:
@mosn Could you clarify why we need to change `IPHONEOS_DEPLOYMENT_TARGET` to 10.0?